### PR TITLE
Replay commits

### PR DESCRIPTION
**Pull request by ssk2**
Bringing this repo up to speed with the private fork.

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,385 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+
+	<groupId>com.mesosphere.velocity</groupId>
+	<artifactId>velocity</artifactId>
+	<version>1.2-SNAPSHOT</version>
```
Comment (rji) [inline pom.xml]: Assuming that this is going to be the 1.0.0 release, we should update the version here

File: pom.xml [hunk 2]
```diff
@@ -0,0 +1,385 @@
...
+		<repository>
+			<id>jenkins</id>
+			<url>http://maven.jenkins-ci.org/content/repositories/releases/</url>
+		</repository>
+	</repositories>
+	<dependencies>
+		<dependency>
+			<groupId>org.jenkins-ci.main</groupId>
+			<artifactId>jenkins-war</artifactId>
+			<version>1.658</version>
```
Comment (rji) [inline pom.xml]: This should be the latest non-2.x LTS release

File: pom.xml [hunk 3]
```diff
@@ -0,0 +1,385 @@
...
+				<version>1.3.0</version>
+				<executions>
+					<execution>
+						<id>fetch-marathon-deploy</id>
+						<phase>generate-sources</phase>
+						<goals>
+							<goal>wget</goal>
+						</goals>
+						<configuration>
+							<url>https://s3.amazonaws.com/downloads.mesosphere.io/velocity/jenkins/plugins/marathon-1.2.0.hpi</url>
```
Comment (rji) [inline pom.xml]: This is available in the Jenkins update center now

**Comment by rji** [comment]
A couple of inline comments, plus we shouldn't need `scripts/plugin_install.sh` since we moved to Maven for plugin management.

File: CHANGELOG.md [hunk 4]
```diff
@@ -1,4 +1,7 @@
 # Changes
+## v1.0.0 (2016-03-24)
```
Comment (rji) [inline CHANGELOG.md]: We should probably make the v1.0.0 release date today's date (2016-07-07)

Comment (ssk2) [inline CHANGELOG.md]: Done (will force push up in a minute).

File: pom.xml (hunk 2 above)
Comment (ssk2) [inline pom.xml]: Why is this 1.658 out of interest? Looks like the most recent stable was 1.651.3.

File: pom.xml [hunk 5]
```diff
@@ -0,0 +1,393 @@
...
+				</configuration>
+			</plugin>
+
+			<plugin>
+				<groupId>com.googlecode.maven-download-plugin</groupId>
+				<artifactId>download-maven-plugin</artifactId>
+				<version>1.3.0</version>
+				<executions>
+					<execution>
+						<id>fetch-marathon-deploy</id>
```
Comment (rji) [inline pom.xml]: This is in the Jenkins update center (version 1.3.1) so we can treat it as any of the other plugins now and not host it in our S3 bucket

File: pom.xml [hunk 6]
```diff
@@ -0,0 +1,393 @@
...
+		<repository>
+			<id>jenkins</id>
+			<url>http://repo.jenkins-ci.org/releases/</url>
+		</repository>
+	</repositories>
+	<dependencies>
+		<dependency>
+			<groupId>org.jenkins-ci.main</groupId>
+			<artifactId>jenkins-war</artifactId>
+			<version>1.651.2</version>
```
Comment (rji) [inline pom.xml]: Can we bump this to 1.651.3?

File: README.md [hunk 7]
```diff
@@ -15,33 +15,50 @@ up and running quickly with Jenkins on DCOS.
 
 ## Included in this repo
 Base packages:
-  * [Jenkins][jenkins-home] 1.642.4 (LTS)
+  * [Jenkins][jenkins-home] 1.651.2 (LTS)
```
Comment (rji) [inline README.md]: Can we bump this to 1.651.3?

File: scripts/release.sh [hunk 8]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -o errexit -o nounset -o pipefail
+
+# vars
+readonly DOCKER_IMAGE="mesosphere/velocity"
```
Comment (rji) [inline scripts/release.sh]: `s/velocity/jenkins/`

File: scripts/release.sh [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+docker save --output="${TAR_FILE}" $DOCKER_IMAGE:$TAG
+
+# check integrity
+tar tf $TAR_FILE > /dev/null
+# gzip with best compression
+gzip -9 $TAR_FILE
+
+# generate sigs
+openssl md5 $TAR_FILE.gz >> velocity-$TAG.checksums
+openssl sha1 $TAR_FILE.gz >> velocity-$TAG.checksums
```
Comment (rji) [inline scripts/release.sh]: Same for these two lines: `s/velocity/jenkins/`

File: scripts/release.sh [hunk 10]
```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+set -o errexit -o nounset -o pipefail
+
+# vars
+readonly DOCKER_IMAGE="mesosphere/velocity"
+readonly TAG=${GIT_BRANCH##*/}
+readonly TAR_FILE="velocity-${TAG}.tar"
```
Comment (rji) [inline scripts/release.sh]: `s/velocity/jenkins/`

File: .dockerignore [hunk 11]
```diff
@@ -3,4 +3,5 @@
 README.md
 demo/
 docs/
+scripts/build.sh
```
Comment (rji) [inline .dockerignore]: It looks like the script we want to ignore is actually `scripts/release.sh`.

**Comment by rji** [comment]
Before I forget, we should also take a look at any changes to the package on the private side and submit those to the public Universe (after we tag and push v1.0.0 here, of course)

File: pom.xml [hunk 12]
```diff
@@ -0,0 +1,379 @@
...
+						</artifactItem>
+						<artifactItem>
+							<groupId>org.jenkins-ci.plugins</groupId>
+							<artifactId>mailer</artifactId>
+							<version>${mailer.version}</version>
+							<type>hpi</type>
+							<outputDirectory>${project.build.directory}/plugins</outputDirectory>
+						</artifactItem>
+						<artifactItem>
+							<groupId>org.jenkins-ci.plugins.workflow</groupId>
```
Comment (colin-msphere) [inline pom.xml]: Should just be `org.jenkins-ci.plugins`

File: pom.xml [hunk 13]
```diff
@@ -0,0 +1,379 @@
...
+						<artifactItem>
+							<groupId>org.jenkins-ci.plugins</groupId>
+							<artifactId>ssh-slaves</artifactId>
+							<version>${ssh-slaves.version}</version>
+							<type>hpi</type>
+							<outputDirectory>${project.build.directory}/plugins</outputDirectory>
+						</artifactItem>
+						<!-- Dependencies for Additional Plugins -->
+						<artifactItem>
+							<groupId>org.jenkins-ci.plugins</groupId>
```
Comment (colin-msphere) [inline pom.xml]:
```
[INFO] Configured Artifact: org.jenkins-ci.plugins:icon-shim:2.0.3:hpi
Downloading: http://repo.jenkins-ci.org/public/org/jenkins-ci/plugins/icon-shim/2.0.3/icon-shim-2.0.3.hpi
Downloading: http://repo.jenkins-ci.org/releases/org/jenkins-ci/plugins/icon-shim/2.0.3/icon-shim-2.0.3.hpi
Downloading: https://repo.maven.apache.org/maven2/org/jenkins-ci/plugins/icon-shim/2.0.3/icon-shim-2.0.3.hpi
[ERROR] Failed to execute goal org.apache.maven.plugins:maven-dependency-plugin:2.10:copy (default) on project jenkins: Unable to find artifact. Could not find artifact org.jenkins-ci.plugins:icon-shim:hpi:2.0.3 in repo.jenkins-ci.org (http://repo.jenkins-ci.org/public/)
```

**Comment by rji** [comment]
I'd like to do some manual testing on this tomorrow before merging, but otherwise this looks good to me.

**Comment by ssk2** [comment]
🚀

**Pull request merged**